ERCS/erc-7629.md

```diff
@@ -0,0 +1,296 @@
+---
+eip: 7629
+title: Unified Token
+description: Unifying ERC-721 and ERC-20 for seamless asset conversion.
+author: 0xZeus1111 (@0xZeus1111), Nvuwa (@Nvuwa)
+discussions-to: https://ethereum-magicians.org/t/erc-7629-unified-token-protocol/18793
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-02-18
+requires: 20, 721
```

Comment:
```suggestion
requires: 20
```
Requies only the 20 for full compatibility, meaning should inherit IERC20 maybe?

Reply:
ERC-721 is also required, with ERC-20 being fully compatible and ERC-721 partially compatible.